app/controllers/errors_controller.rb

```diff
@@ -4,6 +4,12 @@ class ErrorsController < ApplicationController
   layout "errors"
 
   def not_found
+    Bugsnag.notify("404") do |event|
+      event.severity = "info"
+
+      event.add_metadata(:request, request.env)
+      event.add_metadata(:params, params)
```

Comment:
Did you test this?

Reply:
I saw there were a tab `request` and I thought that was something related to my code but maybe not... I should have test without those lines maybe...

Reply:
Only request is enough. Modified then.


Moving to `Test Ready`